news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,208 @@
+
+If you are interested in all previous and future patch notes, you can find them [here](https://osu.ppy.sh/home/changelog).
+
+One thing you might've missed is the new fancy Repel mod added to lazer, which is pretty chaotic but fun (finally a worthy opponent for [badeu](https://osu.ppy.sh/users/1473890)?)
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://user-images.githubusercontent.com/47010459/172435116-6b09a48c-6fd6-4abb-9e5f-e96ab3565184.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+Another feature that's maybe not the newest, but some of you might have missed, or just haven't gotten around to trying yet is the **skin layout editor**! First implemented in May of 2021, this is still very much a work-in-progress feature, however currently you are able to edit the placement of all skin elements that appear during gameplay! Check out the showcase video from the original patch notes below:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://user-images.githubusercontent.com/191335/118155608-5c5fce80-b453-11eb-88f4-e552b10ed5f1.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+## Month's Community Quiz
+
+A monthly question or puzzle for you!
+
+> Once upon a time, in the ye' old osu!, someone asked for advice from the ancestors among osu!, the post reads as follows: 
+
+> "Is it possible to repeat songs in osu!? I have an in-game video of KH2 with Roxas battling the Twilight Thorn (people who have played Kingdom Hearts 2 will be drooling) and the video is still halfway when the song ends! Or can someone at least tell me about a free, easy to use software which can solve my problem by repeating the song?"
+
+> This ancient tome is hidden behind a code, the code you seek is everywhere but no where at the same time, always in the back of your mind but only breaching the surface at the moment you see it, now... go forth, unveil the secret code, and paste the number in the link, and gain access to this ancient tome!
+
+> The first person to answer it correctly will receive a month of osu!supporter, leave your answer in the comment!
+
+## End Word
+
+We wanted to thank the osu!team and the contributors to [osu!lazer](https://github.com/ppy/osu), [osu!web](https://github.com/ppy/osu-Web) and [osu!wiki](https://github.com/ppy/osu-wiki) for their hard work on the repositories and the trust thay gave us. Huge thanks again to [Nelly](https://osu.ppy.sh/users/4741164) for the osu!catch news bulletin, [fajar13k](https://osu.ppy.sh/users/7100002) for his contribution to the EGTS part, and to [The-Last-Cookie](https://github.com/The-Last-Cookie) for his amazing review (a lot of things to correct, right?)!
+
+If you're interested on seeing the creation processus, the backstages of osu!, please join the [osu!dev server](https://discord.gg/ppy)! (Articles from external contributors are welcome)
```

Comment:
``` Suggestion
If you're interested in seeing the creation process, and the backstages of osu!, please join the [osu!dev server](https://discord.gg/ppy)! (Articles from external contributors are welcome)
```

Reply:
Just as an aside note, because it interested me: [processus](https://dictionary.cambridge.org/us/dictionary/english/processus) is a real English word. However, I agree with your suggestion, as it is more readable and I believe that `processus` is not a common word.